src/main/java/com/comphenix/protocol/utility/MinecraftMethods.java

```diff
@@ -231,6 +253,11 @@ private static void initializePacket() {
                 }
             }
 
+            // write and read methods are no longer part of the packet interface since 1.20.5
+            if (MinecraftVersion.v1_20_5.atOrAbove()) {
```

Comment:
do we want to throw an exception here?

Reply:
Yeah I guess we could throw an exception when the read/write method getters get called on 1.20.5 upwards